src/views/AddAccountSuccess.vue

```diff
@@ -1,29 +1,29 @@
 <template>
-    <div class="container">
-        <small-page v-if="$route.name === `add-account-success`">
+    <div class="container pad-bottom">
+        <SmallPage>
             <PageHeader>Your account is ready</PageHeader>
-            <div class="page-body">
-                <div class="success-box">
-                    <h2>Cool!</h2>
-                    <p>Your new account has been added to your wallet.</p>
+            <PageBody>
+                <div class="success-box nq-icon trumpet nq-bg-green">
+                    <h2 class="nq-h2">Awesome!</h2>
```

Comment:
Should we also ride the "Awesome" wave? Alternatives: "Perfect!", "Nice!", or maybe "Delightful!", "Sweet!"

Reply:
Wording is by no means final. Mostly just a placeholder until we figure out a proper wording revision process.